src/platform/user/profile/utilities/index.js

```diff
@@ -108,6 +108,9 @@ export function mapRawUserDataToState(json) {
     userState.status = getErrorStatusDesc(errorStatus);
   } else {
     userState.status = vaProfile.status;
+    if (vaProfile.facilities) {
```

Comment:
@jbalboni can you confirm that right now the `va_profile.facilities` array that we get from the `GET user` endpoint is not being stored in state anywhere? I didn't see it so I added it here.

Reply:
Yeah, I didn't see it and I am doing the same thing you're doing for a ticket I have.

Reply:
so race you? ha, I'm gonna try to get this merged ASAP if that's cool with you

Reply:
Yeah, go for it, I won't have mine up for a little whie.